lld/docs/ReleaseNotes.rst

```diff
@@ -26,6 +26,14 @@ Non-comprehensive list of changes in this release
 ELF Improvements
 ----------------
 
+* Section ``CLASS`` syntax allows binding input section to named classes. This
+  allows the linker to automatically pack the input sections into memory
+  regions by automatically spilling to later class references if a region would
+  overflow. This reduces the toil of manually packing regions (typical for
+  embedded). It also makes full LTO feasible in such cases, since IR merging
```

Comment:
How is full LTO feasible? I would expect that brief items in release notes have some description in the patch descriotion.

Reply:
Thanks, this went out of date now that we've cut the release branch. Reworded.